pkg/ddevapp/backdrop.go

```diff
@@ -76,7 +76,7 @@ func createBackdropSettingsFile(app *DdevApp) (string, error) {
 	} else {
 		output.UserOut.Printf("Existing %s file does not include %s, modifying to include ddev settings", settings.SiteSettings, settings.SiteSettingsDdev)
 
-		if err = appendIncludeToDrupalSettingsFile(app.SiteSettingsPath, app.Type); err != nil {
+		if err = appendIncludeToDrupalSettingsFile(nil); err != nil {
```

Comment:
I didn't test it, but i think `nil` can cause panic here.

```suggestion
		if err = appendIncludeToDrupalSettingsFile(app); err != nil {
```

Reply:
Must be `app` anyway. thanks for the good eye.